tutorials/fundamentals/installation.md

```diff
@@ -1,55 +1,101 @@
-# napari installation tutorial
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.0
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# How to install napari on your machine
+
+Welcome to the **napari** installation guide!
+
+This guide will teach you how to do a clean install of **napari** and launch the viewer.
+
+```{note} 
+If you want to contribute back to the napari codebase, you should follow [this guide instead](https://napari.org/developers/contributing.html).
+```
 
-Welcome to the **napari** installation tutorial!
+## Prerequisites
 
-This tutorial will teach you how to do a clean install of **napari**. It is
-aimed at people that just want to use napari. For people also interested in
-contributing to napari please check our [contributing
-guidelines](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md)
-for more advanced installation procedures. At the end of the tutorial you should
-have napari successfully installed on your computer and be able to make the
-napari viewer appear.
+Prerequisites differ depending on how you want to install napari.
 
-## Installation
+### Installing as a Python package 
+This installation method allows you to use napari from Python to programmatically 
+interact with the app. It is the best way to install napari and make full use of
+all its features.
 
-### Which distribution to install
+It requires:
+- [Python 3.7+](https://www.python.org/downloads/)
```

Comment:
```suggestion
- [Python 3.7 or higher](https://www.python.org/downloads/)
```

Just for clarity :smile:

Reply:
Although, I'm not sure you should link to https://www.python.org/downloads/ if you are later going to try to encourage the use of conda. Maybe the link can be removed?